src/components/Toast/Toast.tsx

```diff
@@ -47,7 +47,6 @@ export class Toast extends React.PureComponent<ComposedProps, never> {
         message: content,
         duration,
         isError: error,
-        isDismissible: true,
```

Comment:
Just to be clear, calling app bridge toast now always has a dismissible (`x`) button?

Reply:
Found the answer